packages/cozy-clisk/src/launcher/saveFiles.js

```diff
@@ -64,7 +62,7 @@ const saveFiles = async (client, entries, folderPath, options) => {
     throw new Error('Savefiles : No list of files given')
   }
   if (entries.length === 0) {
-    log.warn('No file to download')
+    options.log('warn', 'saveFiles', 'No file to download')
```

Comment:
do we always have this log from option?

Reply:
This saveFiles is called by the flagship app and cozy-clisk is included in the flagship app.Another PR will come with the update of cozy-clisk and `log` option.

Reply:
Yep, but in reality this change is a breaking change. You have to edit the caller code in order to make it work without breaking (aka here throwing an error `options.log is not a function`). 

So tag it as a BC or make it optional by checking if these is an `options.log` before calling it

Reply:
Then made it optionnal